b2share/modules/schemas/serializers.py

```diff
@@ -60,7 +60,10 @@ def block_schema_version_to_dict(block_schema_version):
 
 def block_schema_version_to_json_serializer(block_schema_version, code=200,
                                             headers=None):
-    response = jsonify(block_schema_version_to_dict(block_schema_version))
+    if block_schema_version:
```

Comment:
Is this what you meant?
```python
if block_schema_version is not None:
```
`None` is not the only value which casts to `False` in python.